alphagov/paas-rubbernecker
# Support requesting pairing on stories

## What

Adds a new image containing a heart over a pear, to represent a wish for pairing to take place. Configures this to be displayed on stories with `pairing: please` or `please-pair-on-this` labels set.

## How to review

* Check I've configured it correctly;
* Check the image is cute and intuitive enough.
